rust/openvasd/src/controller/entry.rs

```diff
@@ -187,6 +187,7 @@ where
                                                 .as_ref()
                                                 .unwrap_or(&"".to_string())
                                         );
+                                        progress.clean_sensitive_data();
```

Comment:
I'm not sure if removing credentials immediately after starting is sustainable, as the scanner may need the data later on. Although I am aware that currently the data is duplicated by ospd the moment that changes this call can be easily overlooked and debugging missing credentials on a large scan is very cumbersome.

Instead I would promote encryption using a stream cipher like cha20 for credentials so that it is encrypted in memory and decrypting on a per need basis. Additionally we should remove credentials when a scan is finished or if we want to allow reusage of scans later on delete.

Reply:
I removed this line. So the credentials are still in memory.